website/pizzas/services.py

```diff
@@ -29,3 +31,22 @@ def can_change_order(member, food_event):
         and member.has_perm("pizzas.change_foodorder")
         and is_organiser(member, food_event.event)
     )
+
+
+def execute_data_minimization(dry_run=False):
+    """Anonymizes pizzas orders older than 3 years."""
+    # Sometimes years are 366 days of course, but better delete 1 or 2 days early than late
+    deletion_period = timezone.now().date() - timezone.timedelta(days=(365 * 3))
```

Comment:
Maybe this should be a setting or global thing somewhere? Would that make sense?

Reply:
Let me create a new issue for this, because I like the idea!

Reply:
See #2099